home/import_assessments.py

```diff
@@ -385,16 +353,11 @@ def from_flat(cls, row: dict[str, str], row_num: int) -> "AssessmentRow":
         medium_inflection = row.get("medium_inflection")
         check_punctuation(high_inflection, medium_inflection, row_num)
 
-        try:
-            row = {
-                key.strip(): value.strip()
-                for key, value in row.items()
-                if value and key.strip() in cls.fields()
-            }
-        except AttributeError:
-            raise ImportAssessmentException(
-                "Invalid format. Please check that all row values have headers."
-            )
+        row = {
+            key.strip(): value.strip()
```

Comment:
Do we need to strip again if the helper is doing this for us before we get the data?